src/pages/LandingPage.tsx

```diff
@@ -14,28 +15,28 @@ import {
 
 import MoneyIcon from '@material-ui/icons/Money';
 import LoyaltyIcon from '@material-ui/icons/Loyalty';
-import CastForEducationIcon from '@material-ui/icons/CastForEducation';
-import InfoIcon from '@material-ui/icons/Info';
-import TrendingUpIcon from '@material-ui/icons/TrendingUp';
-import WarningIcon from '@material-ui/icons/Warning';
 
-import Card from './components/Card';
-import Media from './components/Media';
-import GridRowContainer from './components/GridRowContainer';
-import img2 from '../assets/images/media2.png';
 import analyse from '../assets/images/analyse.png';
 import lernen from '../assets/images/lernen.png';
 import portfolio from '../assets/images/Portfolio.png';
 import LogoCard from './components/LogoCard';
+import Card from './components/Card';
+import InfoCard from './components/InfoCard';
+import GridRowContainer from './components/GridRowContainer';
 
 const useStyles = makeStyles((theme: Theme) =>
   createStyles({
+    container: {
+      margin: '0px',
+      padding: '0px',
+      display: 'inline',
+    },
     mediaSmall: {
-      maxWidth: '480px',
-      maxHeight: '396px',
+      maxWidth: '450px',
```

Comment:
Use [theme.spacing](https://material-ui.com/customization/spacing/) to make styles more dynamic